service/src/test/java/com/ge/predix/controller/test/PolicyEvaluationControllerIT.java

```diff
@@ -94,7 +96,6 @@ public void setup() {
         this.zoneService.upsertZone(this.testZone);
         MockSecurityContext.mockSecurityContext(this.testZone);
         MockAcsRequestContext.mockAcsRequestContext(this.testZone);
-
```

Comment:
Four lines above (#93-96) could be replaced with one call to TestUtils:
this.testZone = new TestUtils().setupTestZone(...)